src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -494,6 +496,12 @@
     </ItemGroup>
   </Target>
 
+  <!-- Collect packages that are incompatible with Build Acceleration. -->
+  <PropertyGroup Condition="'$(CollectBuildAccelerationIncompatiblePackageDesignTimeDependsOn)' == ''">
+    <CollectBuildAccelerationIncompatiblePackageDesignTimeDependsOn></CollectBuildAccelerationIncompatiblePackageDesignTimeDependsOn>
+  </PropertyGroup>
+  <Target Name="CollectBuildAccelerationIncompatiblePackageDesignTime" DependsOnTargets="$(CollectBuildAccelerationIncompatiblePackageDesignTimeDependsOn)" Returns="@(BuildAccelerationIncompatiblePackage)" />
```

Comment:
This allows extenders to add more items in targets, should they need to. I expect most additions to `BuildAccelerationIncompatiblePackage` would be done via evaluation though.